django_prometheus/middleware.py

```diff
@@ -130,7 +134,7 @@ def process_response(self, request, response):
         responses_by_status.labels(str(response.status_code)).inc()
         if hasattr(response, 'charset'):
             responses_by_charset.labels(str(response.charset)).inc()
-        if response.streaming:
+        if hasattr(response, 'streaming') and response.streaming:
```

Comment:
Please add a comment at l82 indicating that this export is disabled on django < 1.x.